### PR TITLE
AIQ-5472: Make icon hoverable and clickable

### PR DESCRIPTION
**Pull request by atsunechka**
image

### CODE REVIEW

File: src/SelectMulti.vue [hunk 1]
```diff
@@ -425,7 +429,7 @@
 				</div>
 			</slot>
 			</div>
-			<div class="combo-input-icon-block" :class="{ 'not-clickable': !$slots['input-icon'] }">
+			<component :is="iconIsClickable ? 'button' : 'div'" @click="iconIsClickable ? updateMenuState(iconIsClickable) : false" class="combo-input-icon-block" :class="{ 'not-clickable': !iconIsClickable }">
```
Comment (ivan-shtyrlyaev-nitka) [inline src/SelectMulti.vue:432]: To make it clear that this button is meant to always open 
```suggestion
			<component :is="iconIsClickable ? 'button' : 'div'" @click="iconIsClickable ? updateMenuState(true) : false" class="combo-input-icon-block" :class="{ 'not-clickable': !iconIsClickable }">
```

Comment (ivan-shtyrlyaev-nitka) [inline src/SelectMulti.vue:432]: what does this `not-clickable` css class do ? 
Is it just for tests ? If yes I would prefer to follow practise with `data-testid` attribute

**Review by ivan-shtyrlyaev-nitka — CHANGES_REQUESTED** [review]
In my opinion this is wrong.
The main idea - combobox is an input (complex input with search and dropdown).
And this functionality is unlikely to be used and need a lot.

based on ticket environment we need to add just cursor: pointer for the icon on hover, and we could do it right in agency IQ using :deep. 
Nothing to change in `vue-accessible-selects` package

**Review by ivan-shtyrlyaev-nitka — APPROVED** [review]

**Review by cafloyd — APPROVED** [review]
Looks great @atsunechka , and thanks for your help @ivan-shtyrlyaev-nitka !

@atsunechka - you'll also need to PR to the `next` branch since we also currently have an alpha version of this library. 
(We still need to deprecate Vue 2 support in VAS and move the alpha (Vue 3) version to be  the only version - I'll flag this issue for the FE team, it's one that has been brought up and we haven't dealt with yet.)

**Pull request merged**